crates/uv/src/target.rs

```diff
@@ -0,0 +1,168 @@
+    X8664UnknownLinuxMusl,
+}
+
+impl TargetTriple {
+    /// Return the [`Platform`] for the target.
+    pub(crate) fn platform(self) -> Platform {
+        match self {
+            Self::X8664PcWindowsMsvc => Platform::new(Os::Windows, Arch::X86_64),
+            Self::X8664UnknownLinuxGnu => Platform::new(
+                Os::Manylinux {
+                    major: 2,
+                    minor: 17,
+                },
+                Arch::X86_64,
+            ),
+            Self::X8664AppleDarwin => Platform::new(
+                Os::Macos {
+                    major: 10,
+                    minor: 12,
+                },
+                Arch::X86_64,
+            ),
+            Self::Aarch64AppleDarwin => Platform::new(
+                Os::Macos {
+                    major: 11,
+                    minor: 0,
+                },
+                Arch::Aarch64,
+            ),
+            Self::Aarch64UnknownLinuxGnu => Platform::new(
+                Os::Manylinux {
+                    major: 2,
+                    minor: 17,
+                },
+                Arch::Aarch64,
+            ),
+            Self::Aarch64UnknownLinuxMusl => {
+                Platform::new(Os::Musllinux { major: 1, minor: 2 }, Arch::Aarch64)
+            }
+            Self::X8664UnknownLinuxMusl => {
+                Platform::new(Os::Musllinux { major: 1, minor: 2 }, Arch::X86_64)
+            }
+        }
+    }
+
+    /// Return the `platform_machine` value for the target.
+    pub(crate) fn platform_machine(self) -> &'static str {
+        match self {
+            Self::X8664PcWindowsMsvc => "x86_64",
+            Self::X8664UnknownLinuxGnu => "x86_64",
+            Self::X8664AppleDarwin => "x86_64",
+            Self::Aarch64AppleDarwin => "aarch64",
+            Self::Aarch64UnknownLinuxGnu => "aarch64",
+            Self::Aarch64UnknownLinuxMusl => "aarch64",
+            Self::X8664UnknownLinuxMusl => "x86_64",
+        }
+    }
+
+    /// Return the `platform_system` value for the target.
+    pub(crate) fn platform_system(self) -> &'static str {
+        match self {
+            Self::X8664PcWindowsMsvc => "Windows",
+            Self::X8664UnknownLinuxGnu => "Linux",
+            Self::X8664AppleDarwin => "Darwin",
+            Self::Aarch64AppleDarwin => "Darwin",
+            Self::Aarch64UnknownLinuxGnu => "Linux",
+            Self::Aarch64UnknownLinuxMusl => "Linux",
+            Self::X8664UnknownLinuxMusl => "Linux",
+        }
+    }
+
+    /// Return the `platform_version` value for the target.
+    pub(crate) fn platform_version(self) -> &'static str {
+        match self {
+            Self::X8664PcWindowsMsvc => "",
+            Self::X8664UnknownLinuxGnu => "",
+            Self::X8664AppleDarwin => "",
+            Self::Aarch64AppleDarwin => "",
+            Self::Aarch64UnknownLinuxGnu => "",
+            Self::Aarch64UnknownLinuxMusl => "",
+            Self::X8664UnknownLinuxMusl => "",
```

Comment:
This really doesn't seem to be used: https://github.com/search?type=code&q=%22platform_version%22+path%3Arequirements.txt.

On my machine:

```python
>>> import platform
>>> platform.version()
'Darwin Kernel Version 23.4.0: Fri Mar 15 00:12:37 PDT 2024; root:xnu-10063.101.17~1/RELEASE_ARM64_T6031'
```

The one common case (guessing it's cargo culted) is using `'ARM' in platform_version # Mac M-chips` vs. `'ARM' not in platform_version # Mac Intel chips`.